biome.jsonc

```diff
@@ -25,26 +25,15 @@
     "rules": {
       "recommended": true,
       "complexity": {
-        "noForEach": "off",
-        "noStaticOnlyClass": "off",
-        "noThisInStatic": "off",
+        // We need couple of empty exports to make those files as esm modules
```

Comment:
supposedly, the rule checks if the empty export is necessary or not, see the valid case https://biomejs.dev/linter/rules/no-useless-empty-export/

Reply:
Done